File: DataBaseRoomApp/jroomapp/src/main/java/com/woowrale/jroomapp/usecases/observers/Observer.java
package com.woowrale.jroomapp.usecases.observers;

import io.reactivex.observers.DisposableSingleObserver;

public class Observer<T> extends DisposableSingleObserver<T> {

    @Override
    public void onSuccess(T t) {

    }

    @Override
    public void onError(Throwable e) {

    }

}

File: DataBaseRoomApp/kroomapp/src/main/java/com/woowrale/kroomapp/usecases/observers/Observer.kt
package com.woowrale.kroomapp.usecases.observers

import io.reactivex.observers.DisposableSingleObserver

abstract class Observer<T> : DisposableSingleObserver<T>() {
    override fun onSuccess(t: T) {}
    override fun onError(e: Throwable) {}
}
